cluster/manifests/skipper/configmap-sandbox-tokeninfo-bridge.yaml

```diff
@@ -0,0 +1,32 @@
+{{ if eq .ConfigItems.local_sandbox_bridge "true"}}
```

Comment:
@mikkeloscar is this ok?

Reply:
Maybe we can prefix it with `skipper_` so it's easier to find, same for the other config items, WDYT?

Reply:
It looks fine from the templating PoV

Reply:
I changed the cfg item keys